.github/workflows/c1-v2.yml

```
@@ -0,0 +1,190 @@
+            --deployment-config-name CodeDeployDefault.AllAtOnce \
+            --deployment-group-name ${{ secrets.AWS_CODEDEPLOY_MAIN_GROUP_V2_NAME }} \
+            --file-exists-behavior OVERWRITE \
+            --s3-location bucket=${{ secrets.AWS_S3_DEPLOY_V2_MAIN_BUCKET_NAME }},bundleType=zip,key=deploy.zip \
+            --output text --query 'deploymentId')
+          echo "DEPLOYMENT_ID=$DEPLOYMENT_ID" >> $GITHUB_ENV
+        
+      - name: Wait for deployment to complete
+        run: |
+          echo "Waiting for deployment $DEPLOYMENT_ID to complete..."
+          aws deploy wait deployment-successful --deployment-id $DEPLOYMENT_ID
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)